packages/scxa-experiment-cell-type-heatmap/README.md

```diff
@@ -0,0 +1,14 @@
+# Single Cell Expression Atlas marker gene heatmap
+[![Build Status](https://travis-ci.com/ebi-gene-expression-group/scxa-marker-gene-heatmap.svg?branch=master)](https://travis-ci.com/ebi-gene-expression-group/scxa-marker-gene-heatmap) [![Coverage Status](https://coveralls.io/repos/github/ebi-gene-expression-group/scxa-marker-gene-heatmap/badge.svg?branch=master)](https://coveralls.io/github/ebi-gene-expression-group/scxa-marker-gene-heatmap?branch=master)
+
+## Installing dependencies
+In order to run the application, you must first install the dependencies using `npm`:
+```
+npm install
+```
+
+## Try it out
+Use [webpack-dev-server](https://github.com/webpack/webpack-dev-server) to run the application in your browser:
+```
+npx webpack-dev-server -d
+```
```

Comment:
The README file isn’t necessary since all these instructions are common to all packages. Also notice that the badges for coverage aren’t used anymore and the ‘Installing dependencies’ section is outdate because we now use Lerna.

At some point I’d like to reengineer all the tests and coverage somehow to display accurately, but I guess that Lerna comes with its own shortcomings.